app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragmentViewModel.kt

```diff
@@ -62,6 +66,16 @@ class SuggestedEditsTasksFragmentViewModel : ViewModel() {
             val wikidataResponse = wikidataCall.await()
             editCountsCall.await()
 
+            // TODO: solidify which wikis we want to show image recommendations for.
```

Comment:
Have we finalized the target wikis for this feature?

Reply:
Yes, it will be available on all wikis, but I would prefer to leave (commented-out) code in case we need to add logic to tell which wikis it's enabled on.